pilot/docker/Dockerfile.proxyv2

```diff
@@ -13,7 +13,7 @@ ADD pilot-agent /usr/local/bin/pilot-agent
 # TODO: disabling due to https://github.com/istio/istio/issues/5745
 # RUN \
 # chgrp 1337 /usr/local/bin/envoy /usr/local/bin/pilot-agent && \
-# chmod 2755 /usr/local/bin/envoy /usr/local/bin/pilot-agent
+ RUN chmod 2755 /usr/local/bin/envoy /usr/local/bin/pilot-agent
```

Comment:
Without this change, when using proxyv2 image, I find `/usr/local/bin/envoy` can't be executed as user `istio-proxy`.

Reply:
cc @ostromart

Reply:
This should be ok (it's chgrp causing problems), but please check that you are still getting coredumps.

Reply:
Talked with Martin offline, I think running with v2 test target can help with this problem. Let me try it again and delete this once e2e_pilotv2_v1alpha3 works.